src/en/components/checkbox/base.md

```diff
@@ -1,14 +1,42 @@
 ---
+title: Checkbox
 layout: "layouts/base.njk"
 github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-checkbox
 figma: https://www.figma.com/community/file/1128687821123298228
 permalink: false
-tags: ['checkboxEN', 'header']
+tags: ["checkboxEN", "header"]
 ---
 
-# checkbox
+# {{ title }}
 
-This element will have the content below it
+`<gcds-checkbox>`
+
+_Also called: checklist, check list._
+
+A checkbox is a question with a set of options for one or multiple selections.
 
 {% docLinks locale stage figma github %}
 {% enddocLinks %}
+
+<div class="b-sm text-secondary px-250 py-500 mt-400 mb-500">
```

Comment:
Can we reduce the box padding a little bit? Right now it's pretty big which makes this box take up a big chunk of the page.

Reply:
Oh, I think `mt-400` got applied on the component and I did not noticed